src/sdk/pynni/nni/compression/torch/speedup/compressor.py

```diff
@@ -187,8 +169,11 @@ def speedup_model(self):
         """
         training = self.bound_model.training
         _logger.info("start to speed up the model")
+        conv_prune_dim = detect_mask_prune_dim(self.masks)
+        set_conv_prune_dim(conv_prune_dim)
+
         _logger.info("fix the mask conflict of the interdependent layers")
-        fix_mask_conflict(self.masks, self.bound_model, self.dummy_input)
+        fix_mask_conflict(self.masks, self.bound_model, self.dummy_input, conv_prune_dim=conv_prune_dim)
```

Comment:
is it better to put `detect_mask_prune_dim` in to `fix_mask_conflict`?

Reply:
as we discussed, conv_prune_dim need to be used by infer_shape.py